src/core/configuration.c

```diff
@@ -26,6 +26,7 @@ simulation_configuration pdes_config;
 #define CKPT_FORCED_FULL_PERIOD_KEY 364
 #define ISS_ENABLED                 365
 #define ISS_ENABLED_MPROTECTION     366
+#define MEM_SUPPORT_LOG             367
```

Comment:
Why using a new configuration if there is ISS_ENABLED_MPROTECTION?.

Reply:
deleted MEM_SUPPORT_LOG